danielwrobert/dwr-gatsby
# Update Dependencies

### Summary

Update all major and monor dependencies.

Testes local build before deploying to production to ensure no major issues.

### Test Plan

-   Checkout branch - `git checkout update/all-dependencies`.
-   Run build and local server - `gatsby build && gatsby serve`
-   Vist local URL and confirm everything works as expected.
-   Visit feed URL and confirm RSS feed is still generated.
